content/docs/concepts/flops/overview.md

```diff
@@ -0,0 +1,158 @@
+
+{{< /details >}}
+
+{{< details "Convenient Installation & native CLI" >}}
+  FLOps can be easily set up by cloning its [repository](https://github.com/oakestra/addon-FLOps) and running the docker-compose file on the same machine as the Oakestra Root Orchestrator.
+  See the [FLOps Manuals](/docs/manuals/flops) for concrete instructions.
+
+  The [Oakestra CLI](/docs/getting-started/deploy-app/with-the-cli/#the-oak-cli) supports a [set of commands](/docs/manuals/cli/features/flops-addon/#oak-addon-flops) to work with FLOps.
+{{< /details >}}
+
+{{< details "Designed for Change" >}}
+  FLOps aims to be easily modifiable and extendable by developers and researchers.
+  FLOps employs the following techniques to strive for high code quality and flexibility:
+  - Use state-of-the-art libraries and frameworks
+  - Extendable CLI
+  - Ready-made extendable multi-platform images and services to automate development and evaluation workflows.
+  - Additional base images with optional development flags to speed up the build and execution times.
+  - Enforcement of proper styling and typing via formatters and linters, including CI.
+{{< /details >}}
+
+As a result, individuals with different levels of expertise in FL, automation, DevOps, containerization, and orchestration can benefit from these techniques and perform FL.
+
+{{< callout context="note" title="Naming Explained" icon="outline/info-circle" >}}
+  **FLOps** stands for *Federated Learning Operations*.
+  It is inspired by the field of *MLOps* that enhances ML with DevOps practices.
+
+  We do not claim ownership of this term or name;
+  on the contrary, we want to spread awareness of the benefits of applying DevOps and other techniques to FL/ML to improve the onboarding, usage, and development of the entire field.
+{{< /callout >}}
+
+### Simplified Architecture
+
+{{<svg "flops/flops_overview/">}}
+
+FLOps’ architecture/workflow consists of the following parts:
+- The user interacts with the FLOps API (directly or via the `oak-cli`), the deployed FL tracking server/GUI, and the inference server.
+- The FLOps management suite acts as a dedicated separate control plane as well as storage for container images and trained results, metrics, metadata, and ML models.
+- The ML code that gets FL-augmented and used for training must be provided as a Git repository.
+- The FLOps Management deploys its required components with the help of Oakestra as services on worker nodes.
+- The different computational components (e.g., learners, container image builders, etc.) are scaled up and distributed among various worker nodes.
+
+
+{{< link-card
+  title="FLOps Manuals"
+  description="Explore how to set up and use FLOps yourself"
+  href="/docs/manuals/flops"
+  target="_blank"
+>}}
+
+{{< link-card
+  title="FLOps Source Code"
+  href="https://github.com/oakestra/addon-FLOps"
+  target="_blank"
+>}}
```

Comment:
Maybe you can mention the source code in "setting up FLOps". In concepts, it comes as a distraction